### PR TITLE
Create test.sewt with records pointing to Nest servers.

### PR DESCRIPTION
**Pull request by SkyExploreWasTaken**

### CODE REVIEW

**Review by iostpa — APPROVED** [review]

**Comment by iostpa** [comment]
OH NVM

File: domains/test.sewt.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "I'm trying to connect the domain to my Nest webserver.",
+    "owner": {
+        "username": "SkyExploreWasTaken",
+        "discord": "skyexplorewastaken"
+    },
+    "record": {
+        "A": ["37.27.51.34"],
+        "AAAA": ["2a01:4f9:3081:399c::4"],
+        "TXT": "domain-verification=skyexplorewastaken"
```
Comment (iostpa) [inline domains/test.sewt.json:10]:
```suggestion
        "TXT": ["domain-verification=skyexplorewastaken"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by SkyExploreWasTaken** [comment]
So does it work now ?

**Review by iostpa — APPROVED** [review]
Now its good

**Comment by SkyExploreWasTaken** [comment]
ok thx

**Pull request merged**